posts/inside-rust/2024-08-22-embedded-wg-micro-survey.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Embedded Working Group Community Micro Survey"
+author: James Munns
+team: Embedded Devices Working Group <https://www.rust-lang.org/governance/wgs/embedded>
+---
+
+The [Embedded devices working group] has launched the [2024 Community Micro Survey] starting
+today, and running until **September 19th, 2024**.
+
+**You can take the survey now by [clicking here][2024 Community Micro Survey].**
+
+[Embedded devices working group]: https://www.rust-lang.org/governance/wgs/embedded
+[2024 Community Micro Survey]: https://www.surveyhero.com/c/uenp3ydt
+
+This survey is aimed at gathering information about the community of users who use the Rust Programming Language
+for Embedded Systems, including on microcontrollers. It is being run as a [Micro Survey], run by the same
+Rust Survey Team responsible for the [Annual Rust Survey]. The survey is only offered in the English language.
+
+[Micro Survey]: https://github.com/rust-lang/surveys/blob/main/micro-surveys.md
+[Annual Rust Survey]: https://blog.rust-lang.org/2024/02/19/2023-Rust-Annual-Survey-2023-results.html
+
+We invite you to take this survey even if you have only just begun with Rust on Embedded Systems,
+or have only experimented with it informally. Your responses will help us gather data over time towards
+the adoption of Rust for these systems.
+
+Please help us spread the word by sharing the [survey link][2024 Community Micro Survey] via your social networks,
+at meetups, with colleagues, and in any other community that makes sense to you.
+
+This survey would not be possible without the time, resources, and attention of members of the Survey Working Group,
```

Comment:
It's actually a `Survey Team` (https://github.com/rust-lang/team/blob/master/teams/community-survey.toml) not a WG, even though we get it wrong all the time :laughing:

Reply:
Oops, I copy and pasted it from a previous article:

```
Searching 534 files for "Survey Working" (regex, case sensitive)

~/personal/blog.rust-lang.org/posts/2022-12-05-survey-launch.md:
    2  layout: post
    3  title: "Launching the 2022 State of Rust Survey"
    4: author: The Rust Survey Working Group
    5  description: "Hearing from you about the seventh year of Rust"
    6  ---

~/personal/blog.rust-lang.org/posts/2023-08-07-Rust-Survey-2023-Results.md:
    2  layout: post
    3  title: 2022 Annual Rust Survey Results
    4: author: The Rust Survey Working Group in partnership with the Rust Foundation
    5  release: false
    6  ---

~/personal/blog.rust-lang.org/posts/2023-12-18-survey-launch.md:
    2  layout: post
    3  title: "Launching the 2023 State of Rust Survey"
    4: author: The Rust Survey Working Group
    5  description: "Share your experience using Rust in the eighth edition of the State of Rust Survey"
    6  ---
    .
   25  Please help us spread the word by sharing the [survey link](https://www.surveyhero.com/c/4vxempzc) via your social media networks, at meetups, with colleagues, and in any other community that makes sense to you.
   26  
   27: This survey would not be possible without the time, resources, and attention of members of the Survey Working Group, the Rust Foundation, and other collaborators. Thank you!
   28  
   29  If you have any questions, please see our [frequently asked questions](https://github.com/rust-lang/surveys/blob/main/documents/Community-Survey-FAQ.md).

~/personal/blog.rust-lang.org/posts/inside-rust/2022-06-21-survey-2021-report.md:
    4  author: Nick Cameron
    5  description: "Download a data report on the 2021 annual community survey."
    6: team: The Survey Working Group <https://www.rust-lang.org/governance/teams/community#Survey%20team>
    7  ---
    8  As usual, we conducted an annual community survey in 2021. We previously shared some some highlights and charts

5 matches across 4 files
```

Fixed this article in https://github.com/rust-lang/blog.rust-lang.org/pull/1384/commits/3d827daf1091179159c1630228464401dcbd3c1b tho!

Reply:
Yeah, as I said, we get it wrong all the time :D I think that it was a WG before, but according to the team repository, it's now a team.